### ISSUE TITLE
App gets Closed

### ISSUE THREAD

**Issue by Layap** [harishnandan/Pacman-lvl1#7]
After playing multiple turns of the game, the app gets closed itself.

**Comment by aditya314** [comment]
Can you tell approximately how many time I need to play the game to notice the bug? I didn't get this error.

**Comment by Layap** [comment]
2 or 3 times?

**Comment by aditya314** [comment]
Yes. You're right. I will fix that in the next pull request. There is some null pointer problem with the timer.  Please notify if the problem persists after the pull request is merged.
